prow/config/config.go

```diff
@@ -174,9 +174,27 @@ type Plank struct {
 	// DefaultDecorationConfig are defaults for shared fields for ProwJobs
 	// that request to have their PodSpecs decorated
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
+	// Deprecated, use JobURLPrefixConfig instead
 	// JobURLPrefix is the host and path prefix under
 	// which job details will be viewable
+	// TODO @alvaroaleman: Remove in September 2019
 	JobURLPrefix string `json:"job_url_prefix,omitempty"`
+	// JobURLPrefixConfig is the host and path prefix under which job details
+	// will be viewable. Use `org/repo`, `org` or `*`as key and an url as value
+	JobURLPrefixConfig map[string]string `json:"job_url_prefix_config,omitempty"`
+}
+
+func (p Plank) GetJobURLPrefix(refs *prowapi.Refs) string {
```

Comment:
eh...... it's this a breaking change? We should still fall back to `JobURLPrefix` if `JobURLPrefixConfig` is empty

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L4, /shrug

cc @srinivashegde86